runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/fields/Minutes/Minutes.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react';
+
+import CustomSelect from '../../components/CustomSelect';
+import PeriodDefinition from '../../components/PeriodDefinition';
+import { UNITS, UnitIndex } from '../../constants';
+import DEFAULT_LOCALE_EN from '../../locale';
+import { PeriodType } from '../../types';
+import { MinutesProps } from './Minutes.types';
+
+// eslint-disable-next-line complexity
+const Minutes: FC<MinutesProps> = ({ 
+    value, 
+    setValue, 
+    locale, 
+    className, 
+    disabled, 
+    readOnly, 
+    leadingZero, 
+    clockFormat, 
+    period 
+}) => (
+    <>
+        <PeriodDefinition 
```

Comment:
because It's used very often (about 10 times) in similar circumstances, and almost always require some condition I decided to put the condition inside as a isDisplayed parameter.

imo it simplifies, take a look at this example, reconsider and decide:
1**. solution with <If>'s**
```
<If condition={period === PeriodType.HOUR}>
    <PeriodDefinition 
        locale={locale}
        localeKey='prefixMinutesForHourPeriod'
    />
</If>
<If condition={period !== PeriodType.HOUR}>
    <PeriodDefinition 
        locale={locale}
        localeKey='prefixMinutes'
    />
</If>
<CustomSelect
    {...}
/>
<If condition={period === PeriodType.HOUR}>
    <PeriodDefinition 
        locale={locale}
        localeKey='suffixMinutesForHourPeriod'
    />
</If>
```

3. **solution without <If>'s**
```
<PeriodDefinition 
    isDisplayed={period === PeriodType.HOUR}
    locale={locale}
    localeKey='prefixMinutesForHourPeriod'
/>
<PeriodDefinition 
    isDisplayed={period !== PeriodType.HOUR}
    locale={locale}
    localeKey='prefixMinutes'
/>
<CustomSelect
    {...}
/>
<PeriodDefinition 
    isDisplayed={period === PeriodType.HOUR}
    locale={locale}
    localeKey='suffixMinutesForHourPeriod'
/>
```

Reply:
imo delete `isDisplayed` and just make state or smthing to decide the `localeKey`. If there is no need to display it just use `If` component

Reply:
done